machine/inventec/inventec_d7054q28b/rootconf/sysroot-init/networking.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/sh
```

Comment:
why is the file needed ?

it is a very old version of the ONIE core /etc/init.d/networking.sh

several bugs have been fixed in the core, which are not reflected here.

i would get rid of this and use the core version